src/Microsoft.PowerShell.MarkdownRender/VT100EscapeSequences.cs

```diff
@@ -274,12 +274,12 @@ public class VT100EscapeSequences
     {
         private const char Esc = (char)0x1B;
 
-        private string endSequence = Esc + "[0m";
+        private readonly string endSequence = Esc + "[0m";
```

Comment:
This could be const.

Reply:
These changes should be made in follow-up PR.

Reply:
Issue opened: #13914